README.md

```diff
@@ -0,0 +1,136 @@
+### Notebook Images
```

Comment:
This should be a top level item (one single #): "# Notebook Images". Change other items in the README accordingly.